backend/app/Http/Controllers/MarketplaceController.php

```diff
@@ -54,6 +55,37 @@ public function updateCard($card_id)
         return $this->getCardDetail($card_id);
     }
 
+    public function buyCard($card_id)
+    {
+        //update card user's id with current id
+        $user = auth()->user();
+        $card = Card::find($card_id);
+        if ($card->isUserOwner($user)) {
+            return response()->build("User own the card");
```

Comment:
Proper response syntax would be to do `return response()->build(self::RESPONSE_MESSAGE_ERROR_UNAUTHORIZED,"User does not own the card");`

see `App/Providers/ResponseServiceProvider.php` for info on how the response is built (and the response types are defined in `Controller.php`)